### DISCUSSION TITLE
README Showing

### DISCUSSION THREAD

**Discussion by caioschristino** [filipecarneiro/hugo-bootstrap-theme#12] [General]
Hi filipe.carneiro how are u?
 i'm trying use the hugo-bootstrap-theme/

Before deploy happens. The index page of site show README and i cannot undertand what do change to index as in the site example : https://filipecarneiro.github.io/hugo-bootstrap-theme/

Now, i did just a fork from repo and execute the pipeline. Without changes. 

U can help-me to understand what happen?
Thank u

**Comment by filipecarneiro** [comment]
Oi Caio,

Sorry for the late reply.

I couldn't find the fork on your [repository list](https://github.com/caioschristino?page=1&tab=repositories). Can you point me in the right direction?
